qa/automation/ScoringPortal.Tests.Scenarios/1. Access to Portal/1.1. Access to Career Portal.feature

```diff
@@ -1,6 +1,67 @@
 Feature: 1.1. Access to Career Portal
 
-Scenario: 1.1_01 User can open Career Portal
-	Given 
-	When 
-	Then 

```

Comment:
Could you complete the syntax here? Since the steps are not properly defined.

Reply:
It's old version of the file